ultimatejobweb/tests.py

```diff
@@ -1,3 +1,14 @@
 # from django.test import TestCase
+Import pytest
+from API_FB import Manipulation
 
-# Create your tests here.
+# Test function
+def test_api_urls():
+    """The testing will check if our functions return the required output"""
+    expected_output_urls = [ 'https://www.facebook.com/careers/jobs/1672813472870915/', 'https://www.facebook.com/careers/jobs/712878282607325/', 'https://www.facebook.com/careers/jobs/420212419368641/', 'https://www.facebook.com/careers/jobs/207984144060781/', 'https://www.facebook.com/careers/jobs/773785633436218/', 'https://www.facebook.com/careers/jobs/2597607417128369/', 'https://www.facebook.com/careers/jobs/323705805605018/', 'https://www.facebook.com/careers/jobs/536078547072736/']
+    assert Manipulation.extract_jobs_list()==expected_output_urls,"The tests for URLs were successful"
+
```

Comment:
That list of outputs is very temporary as jobs change all the time.
For testing purposes, it might make sense to copy their HTML page statically so you can mock the tests.

Reply:
I'm not sure about this comment - If I'll copy the HTML page statically it won't do any test since it is the source of my func